railties/lib/rails/generators/rails/app/templates/public/406-unsupported-browser.html

```diff
@@ -59,7 +59,7 @@
   <div class="dialog">
     <div>
       <h1>Your browser is not supported.</h1>
-      <p>Please upgrade your browser to continue.</p>
+      <p>Please upgrade your browser to continue or change your "allow_browser" setting.</p>
```

Comment:
This page is displayed in production as well, so I'm not sure we should be indicating configuration suggestions here

Reply:
I do recognize that downside, but I think the tradeoff is worth it. It's just a generically styled page which someone is likely to style in their own way. It's not uncommon for default error pages generated by frameworks to make mention of things like settings.

I considered trying to make this an ERB template so it could give extra explanation in development, but I didn't think it was worth that level of complexity.

Reply:
We will not show application users a developer-only message, please remove this from this page.